libnmstate/nm/ipv4.py

```diff
@@ -119,3 +126,95 @@ def get_ip_profile(active_connection):
     if remote_conn:
         return remote_conn.get_setting_ip4_config()
     return None
+
+
+def get_route_info():
+    ret = []
+    client = nmclient.client()
+    for active_connection in client.get_active_connections():
+        if active_connection is None:
+            continue
+        ip_cfg = active_connection.get_ip4_config()
+        if not ip_cfg:
+            continue
+
+        devs = active_connection.get_devices()
+        iface_name = None
+        # If we have master device, we use it, else we use first device.
```

Comment:
This might be valid for Nmstate but in general I believe a profile could be active for multiple interfaces now. Not sure what kind of static routes this would result in.

Reply:
Let's address that in another PR.